src/yb/rocksdb/util/io_posix.cc

```diff
@@ -474,11 +486,18 @@ Status PosixWritableFile::Allocate(uint64_t offset, uint64_t len) {
         fd_, fallocate_with_keep_size_ ? FALLOC_FL_KEEP_SIZE : 0,
         static_cast<off_t>(offset), static_cast<off_t>(len));
   }
-  if (alloc_status == 0) {
-    return Status::OK();
-  } else {
-    return STATUS_IO_ERROR(filename_, errno);
+
+  if (alloc_status != 0) {
```

Comment:
Same check is repeated in the PR.

Can you extract the check into a helper method ?

Reply:
Done! Hopefully the modification to headers/includes and the name of the helper method is not too bad!